README.md

```diff
@@ -116,6 +116,16 @@ unless `CXX` is assigned in the call to make, e.g.
 
   $ make CXX=$CXX
 
+The Makefile has many option variables with default values. These can be used
+by editing Makefile.conf that `make config-...` creates (see above), or by
```

Comment:
Alternative phrasing: The Makefile has many variables influencing the build process. These can be adjusted by modifying the Makefile.conf file which is created at the `make config-...` step (see above), or they can be set by passing an option to the `make` command directly.